modules/tinymce/src/core/main/ts/api/util/Delay.ts

```diff
@@ -119,7 +122,10 @@ const Delay: Delay = {
 
   /**
    * Sets a timer in ms and executes the specified callback when the timer runs out.
+   * <br>
+   * <em>`setTimeout` has been deprecated and marked for removal in TinyMCE 6.0.</em>
```

Comment:
```suggestion
   * <em><code>setTimeout</code> has been deprecated and marked for removal in TinyMCE 6.0. Use the native browser <code>setTimeout</code> API instead.</em>
```
Same fixes as previously and we should also mention the replacement here.